js/index.js

```diff
@@ -1,7 +1,64 @@
+var canvas = document.querySelector('canvas')
+var ctx = canvas.getContext('2d')
+let intervalId
 window.onload = () => {
   document.getElementById('start-button').onclick = () => {
-    startGame();
-  };
+    startGame()
+  }
+}
+const road = new Image()
+road.src = './images/road.png'
 
-  function startGame() {}
-};
+const backgroundImage = {
+  img: road,
+  y: 0,
+  speed: -1,
+
+  draw: function() {
+    this.y++
+    if (this.y < -canvas.width) {
+      this.y = 0;
+    }
+    // cuando la imagen se salga del canvas
+    ctx.drawImage(this.img, 0, this.y, canvas.width, canvas.height)
+    ctx.drawImage(this.img, this.y +canvas.width, 0, canvas.width, canvas.height)
```

Comment:
Aca tambien parece que solo pegaron lo que hicimos en clase, piensen en la logica, si quieren que se mueva sobre el eje vertical lo que tienen que modificar es la y, no la x. Y deben restar y no sumar 😄 
```suggestion
    ctx.drawImage(this.img, 0, this.y - canvas.width,canvas.width, canvas.height)
```